upload.sh

```diff
@@ -2,7 +2,24 @@
 
 set +x # Do not leak information
 
-RELEASE_NAME="continuous" # Do not use "latest" as it is reserved by GitHub
+# did the user request that this no replace their latest CI binary (called
+# contuous) - for example because this is a feature test branch?
+# or is this a tag (and therefore a release)?
```

Comment:
Please check that whole sentence, I don't really understand it

Reply:
Awesome. That got messed up. Will fix.